generators/kubernetes/files.js

```diff
@@ -11,8 +11,8 @@ function writeFiles() {
                 this.app = this.appConfigs[i];
                 this.template('_deployment.yml', `${appName}/${appName}-deployment.yml`);
                 this.template('_service.yml', `${appName}/${appName}-service.yml`);
-
-                if (this.app.prodDatabaseType) {
+                // If we choose microservice with no DB, it is trying to move _no.yml as prodDatabaseType is getting tagged as 'string' type
+                if (this.app.prodDatabaseType !== 'no') {
```

Comment:
I think it might be better to & this with the original as we dont want to move the file when the property is undefined as well. @PierreBesson am I right or is there no way for this to be undefined?